apps/docs/docs/message-db/access-strategies.md

```diff
@@ -1,18 +1,23 @@
 # Access Strategies
 
+An access strategy controls how your category will _access_ events from the
```

Comment:
I hate undocumentation sentences

The access strategy allows you to opt into custom loading or reloading algorithms. For almost all categories in almost all most systems, `Unoptimized` is a good choice; you can always switch later.